staging/src/k8s.io/api/core/v1/generated.proto

```diff
@@ -2666,6 +2666,7 @@ message PodAffinityTerm {
   // +optional
   optional k8s.io.apimachinery.pkg.apis.meta.v1.LabelSelector labelSelector = 1;
 
+  // Deprecated: please use the NamespaceSelector, this field will be removed in future.
```

Comment:
Do we have deprecation plan may I ask?

Reply:
@resouer I am not sure, @bsalamat any thougts ? do we have a deprecation progress?

Reply:
oh, seems we have the deprecation policy here ：
https://kubernetes.io/docs/reference/deprecation-policy/
This is and v1 api upate, according to the policy:
GA: 1 year or 2 releases (whichever is longer)